doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -221,3 +221,38 @@ Execute and clear                     :kbd:`Ctrl` + :kbd:`Enter`
 Execute selection                     Select code, then :kbd:`Ctrl` + :kbd:`Enter`
 ===================================== =============================================
 ```
+
+
+## _Animation Editor_ Timeline ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover over the timeline of the _Animation Editor_.
```

Comment:
It's enough just to be over the AnimationEditor isn't it, not the timeline specifically?

Reply:
Having used the animation editor for animating the gaffer bot, I can confirm both that this would make way more sense if it was enough to just be over the AnimationEditor, but that it currently fails unless you are over the timeline.

Reply:
Which operations are failing for you specifically?

Reply:
`i` and `f` with the cursor over the channel listing are the most obvious cases that don't work.

Reply:
Oh, I took "timeline" to mean the bottom axis of the graph, with the frame numbers, not the entire AnimationGadget frame...

Reply:
I was confused by that, as well. 

I do agree that `i` and `f` should also work when hovering over the PathListingWidget and not just the AnimationGadget. Maybe we need better names to distinguish between the main Gaffer timeline and the AnimationGadget in the AnimationEditor...

Reply:
> I do agree that i and f should also work when hovering over the PathListingWidget and not just the AnimationGadget.

To me it makes sense that `i` and `f` only work in the main pane, and not in the listing, because they're separate widgets.

> Maybe we need better names to distinguish between the main Gaffer timeline and the AnimationGadget in the AnimationEditor...

Yeah, just now I think all we need to do is replace "timeline of the AnimationEditor" with something less confusing. Any suggestions? Curves view? Graph panel? Right hand panel?

Reply:
> To me it makes sense that i and f only work in the main pane, and not in the listing, because they're separate widgets.

I'm trying to figure out if we're miscommunicating somehow, because this feels really obviously wrong to both me and Matti.  It makes so much more sense if the Animation Editor hotkeys work in the whole Animation Editor tab.  It's a really common thing to want to do to select a curve from the Animation Editor's channel listing and then immediately frame it or insert a key.

Reply:
No, I don't think we're miscommunicating - just have different expectations. I don't feel strongly about it at all though - happy to be ignored. I do think we need to fix the comment about "the timeline of the _Animation Editor_" though, because that is confusing.

Reply:
If we're OK with saying that the fact that the hotkeys don't work while over the channel selector is temporary bug, then the docs could just say that you must hover over the _Animation Editor_

Reply:
Sold